tempesta_fw/cfg.c

```diff
@@ -1517,61 +1530,69 @@ tfw_cfg_start_mods(const char *cfg_text, struct list_head *mod_list)
 
 err_recover_stop:
 	TFW_DBG2("stopping already stared modules\n");
-	MOD_FOR_EACH_REVERSE_FROM_CURR(tmp_mod, mod, mod_list) {
+	MOD_FOR_EACH_REVERSE_FROM_CURR(tmp_mod, mod, &tfw_cfg_mods) {
 		mod_stop(tmp_mod);
 	}
 
 err_recover_cleanup:
-	MOD_FOR_EACH_REVERSE(mod, mod_list) {
+	MOD_FOR_EACH_REVERSE(mod, &tfw_cfg_mods) {
 		spec_cleanup(mod->specs);
 	}
 
 	return ret;
 }
 
+int
+tfw_cfg_start(void)
```

Comment:
Function is small and pretty, but it's a new public function. More over, we have restrictions here: it cannot be called twice without cleanup. That's your opinion, should we document it anyway?